gensrc/proto/internal_service.proto

```diff
@@ -740,6 +740,26 @@ message PUpdateTransactionStateResponse {
     repeated StatusPB results = 1;
 }
 
+message PColumn {
+    optional int32 slot_id = 1;
+    optional int64 data_size = 2;
+    optional bytes data = 3;
+}
+
+message PLookUpRequest {
+    optional PUniqueId query_id = 1;
+    optional int32 lookup_node_id = 2;
+    optional int32 request_tuple_id = 3;
+    repeated PColumn request_columns = 4;
+    repeated PSlotDescriptor lookup_slots = 5;
```

Comment:
can you add some comment to explain which fields are row_source_id, scan_range_id, row_id respectively? 

Is PColumn.slot_id is row_id?  if the row_ids sent to each BE is very sparse, how to batch these sparse row_ids？